### PR TITLE
sidekiq

### LINKED ISSUE #35: configure sidekiq

### PR DESCRIPTION
**Pull request by ibawt**
@qq99 we have background job support now.  As well as a redis instance.

closes https://github.com/ibawt/eightfoot/issues/35

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -6,6 +6,10 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(:default, Rails.env)
 
+redis = Redis.new(:host => '127.0.0.1', :port => ENV['BOXEN_REDIS_PORT'] || 6379)
+
+$redis = Redis::Namespace.new(:eightfoot, redis: redis)
```
Comment (kmcphillips) [inline config/application.rb]: What's this global for? Globals are nasty.

Comment (ibawt) [inline config/application.rb]: Is there a better place for it?  We have it in a global in Shopify as well.  It's just in Shopify.redis

Comment (ibawt) [inline config/application.rb]: I guess I can stuff it in Eightfoot, didn't know that was a thing.

Comment (kmcphillips) [inline config/application.rb]: Well there it's not a `$` global, it's attached to an object. Using `$` variables is pretty bad ruby style, that's all.

Comment (kmcphillips) [inline config/application.rb]: Yup. That sounds fine.

**Pull request merged**